src/botPage/bot/TradeEngine/state/reducers/index.js

```diff
@@ -1,15 +1,16 @@
 import * as constants from '../constants';
 
 const initialState = {
-    scope         : constants.STOP,
+    scope: constants.STOP,
     proposalsReady: false,
 };
 
+// eslint-disable-next-line default-param-last
 const signal = (state = initialState, action) => {
```

Comment:
```suggestion
const signal = (action, state = initialState) => {
```
Also you need to change the calls to this function according to the new param list.

Reply:
this is the previous commit. in the lattest commit I reverted it again because it made some problem.